templates/addEvent.phtml

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Add Event</title>
+        <link rel="stylesheet" type="text/css" href="css/genericStyles.css"/>
+        <link rel="stylesheet" href="css/addEventPage.css">
+    </head>
+    <body>
+        <main class="addEventPageWrapper">
+            <h2 id="heading">Add Event</h2>
+            <form action="/addEvent" method="POST" id="addEventForm">
+                <div class="eventDataRow">
+                    <label for="eventName" class="eventLabel">Event Name: </label>
+                    <input type="text" name="eventName" id="eventName" class="eventInput">
+                </div>
+                <div class="eventDataRow">
+                    <label for="location" class="eventLabel">Location: </label>
+                    <input type="text" name="location" id="location" class="eventInput">
+                </div>
+                <div class="eventDataRow">
+                    <label for="eventType" class="eventLabel">Type of Event: </label>
+                    <input type="text" name="eventType" id="eventType" class="eventInput">
+                </div>
+                <div class="eventDataRow">
+                    <label for="date" class="eventLabel">Date: </label>
+                    <input type="date" name="date" id="date" class="eventInput">
+                </div>
+                <div class="eventDataRow">
+                    <label for="startTime" class="eventLabel">Start Time: </label>
+                    <input type="text" name="startTime" id="startTime" class="eventInput" placeholder="hh:mm">
+                </div>
+                <div class="eventDataRow">
+                    <label for="endTime" class="eventLabel">End Time: </label>
+                    <input type="text" name="endTime" id="endTime" class="eventInput" placeholder="hh:mm">
```

Comment:
If you choose to use type date on the above field, you could consider using type time here

Reply:
Solved in ee284a2

Reply:
Date field switched with js plugin, this field will be regex validated later.